.github/workflows/pr_agent.yml

```
@@ -0,0 +1,24 @@
+name: AI PR Review
+
+on:
+  pull_request:
+
+jobs:
+  review:
+    runs-on: ubuntu-latest 
+    steps:
+      - uses: actions/checkout@v3
```

[ReviewCategory.STYLE] The `actions/checkout@v3` action is outdated and uses an end-of-life Node.js version. Please update to `v4` for security and performance improvements.

```suggestion
      - uses: actions/checkout@v4
```